doc/lua_api.txt

```diff
@@ -2385,6 +2386,10 @@ and `minetest.auth_reload` call the authetification handler.
     * Convert a vector to a wallmounted value, used for `paramtype2="wallmounted"`
 * `minetest.wallmounted_to_dir(wallmounted)`
     * Convert a wallmounted value back into a vector aimed directly out the "back" of a node
+* `minetest.dir_to_yaw(dir)`
+    * Convert a vector into a yaw (angle)
+* `minetest.yaw_to_dir(yaw)`
+    * Convert yaw (angle) to a vector
```

Comment:
surely it would be nice to have all that geometric functions like `to_spherical` and `to_cartesian`, cross product, mixed product, etc., but there are way more such functions than could be included in the core API.
@sapier `pitch_to_dir` would be weird: while `yaw_to_dir` can just return vector in the horizontal XZ plane, which (vertical) plane should `pitch_to_dir` choose?